src/app/page.tsx

```diff
@@ -27,34 +32,68 @@ export default function Home() {
     const password = passwordElement.value
     passwordElement.value = ''
 
-    murmurClient.authenticate(username, password).then(() => {
-      setUsername(username)
-      console.log("we assume the result is valid for now, so lets pretend you have a cookie containing the seed");
-      // immediately inspect to see if a proxy already exists for the given username
-      murmurClient.inspect(username).then((address) => {
-        if (address) {
-          setAddress(address.address)
-        } else {
-          console.log(`no proxy found with name ${username}`)
-        }
-      })
+    setUsername(username)
+    murmurClient.authenticate(username, password).then(async () => {
+      await handleInspect(username)
     })
   }
 
   const handleNew = async () => {
-    murmurClient.new(100).then(result => {
-      murmurClient.inspect(username).then((address) => {
-        if (address) {
-          setAddress(address.address)
-        } else {
-          console.log(`no proxy found with name ${username}`)
-        }
-      })
+    murmurClient.new(300, async (result) => {
+      if (result.status.isInBlock) {
+        await handleInspect(username)
+      }
+    });
+  }
+
+  const handleExecuteBalanceTransfer = async () => {
+    murmurClient.inspect(to).then(async account => {
+      // console.log(amount)
+      let balance = new BN(amount * Math.pow(10, 12));
+      if (account) {
+        let tx = murmurClient.getApi()
+          .tx
+          .balances
+          .transferKeepAlive(account.address, balance);
+
+        await murmurClient.execute(tx as Extrinsic, async (result) => {
+          if (result.status.isInBlock) {
+            console.log(
+              `Transaction included at blockHash ${result.status.asInBlock}`
+            )
+            await handleInspect(username)
+          } else if (result.status.isFinalized) {
+            console.log(
+              `Transaction finalized at blockHash ${result.status.asFinalized}`
+            )
+          }
```

Comment:
This is already logged by `murmur.js` https://github.com/ideal-lab5/murmur.js/blob/d8d93e05d173401fe8a7f9a5892de3adc8079a6e/src/MurmurClient.ts#L180-L185